documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -43,7 +43,7 @@ EOF
 ----
 <1> Specify the vCenter user or the {rhv-short} {manager} user.
 <2> Specify the user password.
-<3> {rhv-short} only: Specify the base64-encoded CA certificate of the {manager}. You can retrieve it at `https://<{rhv-short}_engine_host>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA`.
+<3> {rhv-short} only: Specify the CA certificate of the {manager}. You can retrieve it at `https://<{rhv-short}_engine_host>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA`.
 <4> VMware only: Specify the vCenter SHA-1 fingerprint.
```

Comment:
@mguetta1 -- I just noticed that for upstream, {rhv-short} is not going to render as RHV.  I suggest writing it as RHV, without the attribute. What do you think?

Reply:
@RichardHoch we need the attribute, the RHV upstream project is oVirt.
I think it worked before, I remember that in the previews - {rhv-short} is oVirt

Reply:
Great, then please review as is.

Reply:
Regarding the 'Ansible playbook' in the Hook manifest, I understood from @istein1 that it should be Base64-encoded.
Thanks